File: src/main/java/com/fishercoder/solutions/_1470.java
package com.fishercoder.solutions;

public class _1470 {
    public static class Solution1 {
        public int[] shuffle(int[] nums, int n) {
            int[] result = new int[nums.length];
            for (int i = 0, j = 0; i < n && j < 2 * n; i++, j++) {
                result[j] = nums[i];
                result[++j] = nums[i + n];
            }
            return result;
        }
    }
}

File: cpp/_1470.cpp
//Shuffle the Array

class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
    
        vector<int> v;
         
        for(int i=0;i<n;i++){
            v.push_back(nums[i]);
            v.push_back(nums[n+i]);
        }
        
        return v;
    }
};
